### PR TITLE
commit for avtaar and alert component of CL

### PR DESCRIPTION
**Pull request by rajaranjanmr**

### CODE REVIEW

File: clhome.css [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+.about{
+    text-align: left;
+    color:navy;
+    font-size:1.5rem;
+    padding:1rem;
+    font-weight: 200;
+    /* width:50%;
+    position:absolute;
+    left:200px;
+    top:200px; */
```
Comment (sri-rishi) [inline clhome.css]: remove all comments

File: alerts.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+.greenalert{
+    background-color: #77D970;
+}
+
+.bluealert{
+    background-color: #3DB2FF;
+}
+.yellowalert{
+    background-color: #F9D371;
+}
```
Comment (sri-rishi) [inline alerts.css]: use variable naming for colors

File: alerts.css [hunk 3]
```diff
@@ -0,0 +1,40 @@
+.alert{
+    width:70%;
+    height: auto;
+    padding: 15px;
+
```
Comment (sri-rishi) [inline alerts.css]: remove extra spacing

**Review by sri-rishi — CHANGES_REQUESTED** [review]

File: alerts.css [hunk 4]
```diff
@@ -0,0 +1,40 @@
+.alert{
+    width:70%;
+    height: auto;
+    padding: 15px;
```
Comment (mehulsatardekar) [inline alerts.css]: user rem if it is more than 10px

File: alerts.css [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+}
+.contentIcon{
+    cursor: pointer;
+    padding:1rem;
+}
+.contentItem{
+    padding:1rem;
+}
+.greenalert{
+    background-color: #77D970;
```
Comment (mehulsatardekar) [inline alerts.css]: use CSS variables. to avoid repetition.

**Review by mehulsatardekar — APPROVED** [review]
you can merge the branch after the changes.

**Review by sri-rishi — APPROVED** [review]
Can be merged

**Pull request merged**